runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -617,6 +617,8 @@ J9::Compilation::canAllocateInline(TR::Node* node, TR_OpaqueClassBlock* &classIn
 
    bool generateArraylets = self()->generateArraylets();
 
+   bool areValueTypesEnabled = TR::Compiler->om.areValueTypesEnabled();
```

Comment:
Can I suggest making this a `const`?

Reply:
Added `const` in commit https://github.com/eclipse/openj9/pull/9139/commits/b27c9e6af6c7645dea2b198333184faa14b3df04